### PR TITLE
Set up permanent branch deploys for testnets

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: https://github.com/regen-network/regen-registry/issues/838

Needs: https://github.com/regen-network/registry-server/pull/130

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles 
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -2,4 +2,10 @@
...
+  environment = { REACT_APP_SANITY_TAG="default", REACT_APP_SANITY_DATASET = "staging", REACT_APP_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", REACT_APP_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_SANITY_DATASET = "staging", GATSBY_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", GATSBY_INTERCOM_APP_ID = "kn5di10s", REACT_APP_INTERCOM_APP_ID = "kn5di10s", REACT_APP_LEDGER_CHAIN_ID = "regen-hambach-1", REACT_APP_LEDGER_REST_ENDPOINT = "http://hambach.regen.network:1317", REACT_APP_LEDGER_CHAIN_NAME = "Regen - Hambach Testnet", REACT_APP_BLOCK_EXPLORER = "https://hambach.regen.aneka.io", REACT_APP_EXP_LEDGER = true }
```
Comment (blushi) [inline netlify.toml:5]: After this PR is merged into master, I'll create `redwood` and `hambach` branches.
On `hambach`, the current config for deploy previews won't change while on `redwood`, I'll update it to be the one from `context.redwood` (line 11)

File: web-registry/.env.example [hunk 2]
```diff
@@ -8,6 +8,6 @@ REACT_APP_SANITY_DATASET=
 REACT_APP_SANITY_PROJECT_ID=
 REACT_APP_SANITY_TAG=default
 REACT_APP_LEDGER_CHAIN_ID=
-REACT_APP_LEDGER_REST_ENDPOINT=
```
Comment (blushi) [inline web-registry/.env.example:11]: using the registry-server proxies so no need to have a direct reference to this anymore from the front-end

File: netlify.toml [hunk 3]
```diff
@@ -2,4 +2,10 @@
...
+  environment = { REACT_APP_SANITY_TAG="default", REACT_APP_SANITY_DATASET = "staging", REACT_APP_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", REACT_APP_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_SANITY_DATASET = "staging", GATSBY_API_URI = "https://regen-registry-server-staging.herokuapp.com", GATSBY_RECAPTCHA_SITE_KEY = "6LchxlAaAAAAANjtO3w4b5ihVCL87tYfbe0ltzp9", GATSBY_INTERCOM_APP_ID = "kn5di10s", REACT_APP_INTERCOM_APP_ID = "kn5di10s", REACT_APP_LEDGER_CHAIN_ID = "regen-redwood-1", REACT_APP_LEDGER_REST_ENDPOINT = "http://redwood.regen.network:1317", REACT_APP_LEDGER_CHAIN_NAME = "Regen - Redwood Testnet", REACT_APP_BLOCK_EXPLORER = "https://redwood.regen.aneka.io" }
```
Comment (blushi) [inline netlify.toml:5]: After this PR is merged into master, I'll create `redwood` and `hambach` branches.
On `redwood`, the current config for deploy previews won't change while on `hambach`, I'll update it to be the one from `context.hambach` (line 8)

**Review by haveanicedavid — APPROVED** [review]

**Review by mhagel — APPROVED** [review]
Some nice improvements here. Thank you!

**Review by victordelval — APPROVED** [review]

**Comment by blushi** [comment]
Needs https://github.com/regen-network/registry-server/pull/131 before we can merge

**Pull request merged**